### PR TITLE
tiup: add instructions for the --local option

### PR DESCRIPTION
**Pull request by enkilee**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
fix PR16009 https://github.com/pingcap/docs-cn/pull/16009

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16009
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-component-cluster-template.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ If this option is not specified, the output default template contains the follow
 - Outputs a detailed topology template that is commented with configurable parameters. To enable this option, add it to the command.
 - If this option is not specified, the simple topology template is output by default.
 
+### --local
+
+- Outputs a simple topology template for local cluster which can used directly, parameter `global` can be modify by user needed.
+- This template will create PD service, TiDB service, TiKV service, monitoring service and Gfafana service.
```
Comment (dveeden) [inline tiup/tiup-component-cluster-template.md:34]:
```suggestion
- This template will create PD service, TiDB service, TiKV service, monitoring service and Grafana service.
```

Comment (enkilee) [reply tiup/tiup-component-cluster-template.md:34]: Thanks

**Review by dveeden — APPROVED** [review]

File: tiup/tiup-component-cluster-template.md [hunk 2]
```diff
@@ -28,6 +28,11 @@ If this option is not specified, the output default template contains the follow
 - Outputs a detailed topology template that is commented with configurable parameters. To enable this option, add it to the command.
 - If this option is not specified, the simple topology template is output by default.
 
+### --local
+
+- Outputs a simple topology template for local cluster which can used directly, parameter `global` can be modify by user needed.
```
Comment (lilin90) [inline tiup/tiup-component-cluster-template.md:33]:
```suggestion
- Outputs a simple topology template for the local cluster, which can be used directly, and the `global` parameter can be adjusted as needed.
```

File: tiup/tiup-component-cluster-template.md [hunk 3]
```diff
@@ -28,6 +28,11 @@ If this option is not specified, the output default template contains the follow
...
+- This template will create PD service, TiDB service, TiKV service, monitoring service and Grafana service.
```
Comment (lilin90) [inline tiup/tiup-component-cluster-template.md:34]:
```suggestion
- This template creates a PD service, a TiDB service, a TiKV service, a monitoring service, and a Grafana service.
```

File: tiup/tiup-component-cluster-template.md (hunk 2 above)
Comment (enkilee) [reply tiup/tiup-component-cluster-template.md:33]: Thanks

File: tiup/tiup-component-cluster-template.md (hunk 3 above)
Comment (enkilee) [reply tiup/tiup-component-cluster-template.md:34]: Thanks

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**